src/openms/include/OpenMS/ANALYSIS/ID/SiriusMSConverter.h

```diff
@@ -55,7 +56,7 @@ namespace OpenMS
 
     /// store MS file
     /// @return string (full path to file)
-    static void store(const PeakMap & spectra, const OpenMS::String & msfile);
+    static void store(const PeakMap & spectra, const String & msfile, const std::map<size_t, StringList> & map_precursor_to_adducts);
```

Comment:
document parameter, (should also be clear what happens if an empty map is passed)